client_test/client_test.go

```diff
@@ -104,6 +104,7 @@ var _ = Describe("Client Tests", func() {
 		userlib.KeystoreClear()
 
 		userlib.SymbolicDebug = false
+		userlib.SymbolicVerbose = false
```

Comment:
Nit for future: if `userlib.SymbolicDebug` is set to false, then userlib should automatically assume that `SymbolicVerbose` is set to false too. IMO, the debug flag is whether the debug should be on at all, and the verbose flag is a debug level.

We do not need to update userlib to do this now. I just wanted to get my thought down before I forgot.